.gitmodules

```diff
@@ -1,19 +1,22 @@
-[submodule "libs/include/mavlink/v2.0"]
+[submodule "c_library_v2"]
```

Comment:
I did not know we could actually rename them here. But in this case, why not "mavlink_v2"?

Reply:
Yes. Good one, I'll change it.